public/app/themes/justice/src/patch/js/ccfw-gtm.js

```diff
@@ -1,22 +1,27 @@
-import {Storage} from './ccfw-storage';
+import { Storage } from './ccfw-storage'
+
 /**
  * A dataLayer management script for GTM
  *
  * First step; define some settings
  */
 const CCFW = {
-    gtm: ('; ' + document.cookie).split('; ccfw_gtm_id=').pop().split(';').shift(),
     gtmID: document.getElementById('ccfw-page-banner').getAttribute('data-gtm-id'),
-    canRun: (id) => id.startsWith('GTM') || false,
+    isValidID: () => CCFW.gtmID.startsWith('GTM') || false,
     allowedIds: [],
     selector: {
         all: {
             accept: 'cookie-accept',
             decline: 'cookie-decline'
         },
         moreInfo: 'cookie-more-info',
-        settings: 'js-ccfw-settings-button',
-        toggles: 'ccfw-banner__toggle-slider'
+        settings: 'js-ccfw-settings-button'
+    },
+    jq: {
+        toggles: jQuery('.ccfw-banner__toggle-slider'),
```

Comment:
If toggles is a jQuery object, what do you think about the convention of prefixing it with $? Ie. $toggles

Reply:
I have experienced the convention confusing for new developers. Some think you can only cache a jQuery object if you place a $ at the beginning. This isn't true, JavaScript always passes objects by reference, in effect; creating cache. I would much rather code as the language is intended without the added confusion. This is why I place the object within jq